src/idiomatic/leveraging-the-type-system/borrow-checker-invariants/generalizing-ownership.md

```diff
@@ -0,0 +1,74 @@
+---
+minutes: 10
+---
+
+# Lifetimes and Borrows: the Abstract Rules
+
+Refresher on borrow checker basics.
+
```

Comment:
```suggestion
```

I think the title is sufficiently strong, this sentence does not add much. Let's have more vertical space on the slide instead.